AFQ/viz/utils.py

```diff
@@ -429,10 +443,15 @@ def visualize_tract_profiles(tract_profiles, scalar="dti_fa", min_fa=0.0,
             (tract_profiles["bundle"] == bundle)
         ][scalar].values
         ax.plot(fa, 'o-', color=COLOR_DICT[bundle])
-        ax.set_ylim([min_fa, max_fa])
-        ax.set_yticks([0.2, 0.4, 0.6])
-        ax.set_yticklabels([0.2, 0.4, 0.6])
-        ax.set_xticklabels([])
+
+        if (min_scalar is not None) and (max_scalar is not None):
+            ax.set_ylim([min_scalar, max_scalar])
+
+        if use_fa_ticks:
```

Comment:
Maybe clarify in the docstring that `use_fa_ticks` takes precedent over `min_scalar`/`max_scalar`?